static/js/components/HomePage.jsx

```diff
@@ -16,6 +17,9 @@ const HomePage = () => {
       <div style={{ float: "left", paddingRight: "40px", paddingBottom: "40px" }}>
         <NewsFeed />
       </div>
+      <div style={{ float: "left", paddingRight: "40px", paddingBottom: "40px" }}>
```

Comment:
Styling should be defined in CSS whenever possible.

Reply:
Done